.gitignore

```diff
@@ -5,3 +5,6 @@
 
 # Specific files/dirs to ignore for this pipeline
 */tmp/*
+_targets/*
+3_visualize/out/*
+*.DS_Store
```

Comment:
do you have a Mac now, too?! I thought `*.DS_Store` was a Mac-only artifact

Reply:
I do not - wasn't sure what that was 😂, but added it b/c of the instructions:
> When you're satisfied with your code, open a PR to merge the "three-states" branch into "main". Make sure to add _targets/*, 3_visualize/out/*, and any .DS_Store files to your .gitignore file before committing anything

I can add a comment to the issue to note that that's just for Mac if you like

Reply:
OK, so you don't actually have a `.DS_Store` file that you _**had**_ to ignore, right? You can leave in there - maybe add the note if you think it will be useful to you in the future but no worries about making changes. You can gitignore files that don't exist :)